app-web/__tests__/pages/resourceType.test.js

```diff
@@ -131,11 +132,36 @@ describe('Resource Type Template Page', () => {
     expect(Alert).toBeInTheDocument();
   });
 
+  test('Cards are not duplicating after multiple searches', () => {
```

Comment:
I'm liking this test however I feel like we could still run into this issue with the useSearch function misbehaving. This is because we are mocking out useSearch of course. Have you taken a look through the usesearch hook to ensure it will not create duplications?